crates/subspace-networking/examples/requests.rs

```diff
@@ -35,6 +36,8 @@ async fn main() {
         listen_on: vec!["/ip4/0.0.0.0/tcp/0".parse().unwrap()],
         allow_non_global_addresses_in_dht: true,
         request_response_protocols: vec![GenericRequestHandler::create(|&ExampleRequest| async {
+            sleep(Duration::from_secs(2)).await;
```

Comment:
Come on, we need a simple quick regression test that runs in CI every time, not example that waits for 2 seconds and reproduces the issue only when someone runs it, which, admittedly, isn't very often.